### PR TITLE
Update CHANGELOG format and remove rubocop.yml

### LINKED ISSUE #61: Remove .rubocop.yml with Dangerfile
### :speaking_head: Foreword

Thank for taking the time to fill this bug report fully. Without it we may not be able to fix the bug, and the issue may be closed without resolution.

## :ghost: Brief Description

If the only thing in the `.rubocop/yml` is Dangerfile
Run the latest cookstyle
Remove `.rubpopo.yml`

Issue sprouted from sous-chefs/meta/issues/108. If not applicable then issue should be closed.

### PR DESCRIPTION
**Pull request by MarkGibbons**
### Description

Minor fixes.

### Issues Resolved

#61,  #62 and #63 

### Contribution Check List

- [ ] All tests pass.
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable

### CODE REVIEW

**Comment by kitchen-porter** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">This Pull Request is probably missing tests.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,15 @@
...
 All notable changes to this project will be documented in this file.
 
+## Unreleased
+
+### Changed
+
+- Removed Dangerfile from rubocop.yml
+- Deleted rubocop.yml
+- Ran latest cookstyle to clean up minor style issues
+- Update circleci to run in parallel
```
Comment (damacus) [inline CHANGELOG.md:12]: hey @MarkGibbons we're not actually using CircleCI anymore. We've switched over to using Github actions. 

See the `[ci.yml](https://github.com/sous-chefs/filesystem/blob/master/.github/workflows/ci.yml)` for where it's configured

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,14 @@
 
 All notable changes to this project will be documented in this file.
 
+## Unreleased
+
+### Changed
```
Comment (damacus) [inline CHANGELOG.md:7]:
```suggestion
```

My suggestion would be to remove these sections. IMHO they add unnecessary lines to the CHANGELOG, what do you think?

//cc @Xorima

Comment (Xorima) [reply CHANGELOG.md:7]: I agree, this format is useful for a larger project or for slower release process, but here I think we are moving to a much more rapid release process which should negate the need of large change logs for an entry

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,13 @@
 
 All notable changes to this project will be documented in this file.
 
+## Unreleased
+
+
```
Comment (damacus) [inline CHANGELOG.md:7]:
```suggestion
```

**Pull request merged**